packages/docs/celo-codebase/protocol/optics.md

```diff
@@ -0,0 +1,180 @@
+# Optics: OPTimistic Interchain Communication
+
+Optics is a new design for radically cheaper cross-chain communication without header verification. We expect operating Optics to cut 90% of gas costs compared to a traditional header relay. To accomplish this, we took inspiration from optimistic systems (a la Optimistic Roll-ups). Optics features many of the features we prize in an optimistic mechanism, like public verification, low gas fees, broad participation, but has a slightly different security model. 
+
+Optics will form the base layer of a cross-chain communication network that provides fast, cheap communication for all smart contract chains and rollups. It relies only on widely-available cryptographic primitives (unlike header relays), has a latency of a few hours (rather than an ORU’s one week latency), and imposes only about 120,000 gas overhead on message senders.
```

Comment:
How does this gas amount translate into different chains? Will it always cost this much? What's the current "standard/acceptable" cost?

Note: this doesn't have to be changed/answered for this PR

Reply:
Good question! The efficiency of Optics provides an estimated 90% savings compared to other bridging methods. We can't control gas costs for other chains as they differ with their network costs. @prestwich would be able to address this in more depth if needed.